api/app/models/organization.py

```diff
@@ -26,9 +26,11 @@ class Organization(Base, table=True):
 
     # relationships
     projects: List["Project"] = Relationship(back_populates="organization", sa_relationship_kwargs={"lazy":"dynamic"})
+    personas: List["Persona"] = Relationship(back_populates="organization", sa_relationship_kwargs={"lazy":"dynamic"})
```

Comment:
It seemed like personas should be org-level? like if you are Kellogs and you invent Tony the Tiger persona, you'll want to share that across projects